pkg/agent/openflow/network_policy.go

```diff
@@ -36,6 +38,8 @@ const (
 	MatchUDPDstPort
 	MatchSCTPDstPort
 	Unsupported
+
+	DefaultPriorityStr string = "K8sDefault"
```

Comment:
What about using the actual uint16 value used in the OF entry by default? I don't think this string is meaningful here.

Reply:
Done